e2e/package.json

```diff
@@ -15,5 +15,10 @@
   "license": "ISC",
   "dependencies": {
     "cypress": "^3.1.5"
+  },
+  "devDependencies": {
```

Comment:
❔ I thought using this was a one-time-thing; so, does this need to be added?

Reply:
lol just some shameless advertising by the plugin i spose

Reply:
actually looks like that was standing in for the `webpack-preprocsser` and `ts-loader` deps. so i just added those back in